src/components/selection-list/selection-list-view.tsx

```diff
@@ -0,0 +1,88 @@
+import {observer} from 'mobx-react';
+import React = require('react');
+import {SBComponent} from 'stylable-react-component';
+import {root} from 'wix-react-tools';
+import {noop} from '../../utils';
+import {SelectionListItem, SelectionListItemValue, SelectionListModel} from './selection-list-model';
+import listStyle from './selection-list.st.css';
+
+function closestElementMatching(
+    predicate: (element: HTMLElement) => boolean,
+    startAt: HTMLElement
+): HTMLElement | null {
+    let current: HTMLElement | null = startAt;
+    while (current && !predicate(current)) {
+        current = current.parentElement;
+    }
+    return current;
+}
+
+export interface ViewProps {
+    className?: string;
+    focused?: boolean;
+    list: SelectionListModel;
+    onBlur?: React.FocusEventHandler<HTMLElement>;
+    onChange?: (value: SelectionListItemValue) => void;
```

Comment:
Since we started to use `ChangeEvent` interface for all `onChange` events, we should use it here as well, for the consistency. (`e.value` instead of `value` directly)